ios/Flutter/flutter_export_environment.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
+export "FLUTTER_ROOT=/Users/diogosilva/Development/flutter"
```

Comment:
Same here - Can we remove this file or add it to gitignore since it is using path on your personal laptop?